### PR TITLE
Build e2e from actuator pkg dep

### PR DESCRIPTION
**Pull request by ingvagabund**
Flip to github.com/openshift/cluster-api-actuator-pkg

### CODE REVIEW

**Comment by ingvagabund** [comment]
@spangenberg @frobware PTAL

File: Makefile [hunk 1]
```diff
@@ -76,7 +76,7 @@ integration: ## Run integration test
 
 .PHONY: test-e2e
 test-e2e: ## Run e2e test
-	go test -timeout 20m -v sigs.k8s.io/cluster-api-provider-aws/test/machines -kubeconfig $${KUBECONFIG:-~/.kube/config} -ssh-key $${SSH_PK:-~/.ssh/id_rsa} -cluster-id $${ENVIRONMENT_ID:-""} -ginkgo.v
+	go test -timeout 20m -v sigs.k8s.io/cluster-api-provider-aws/test/machines -kubeconfig $${KUBECONFIG:-~/.kube/config} -ssh-key $${SSH_PK:-~/.ssh/id_rsa} -actuator-image gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1 -cluster-id $${ENVIRONMENT_ID:-""} -ginkgo.v
```
Comment (frobware) [inline Makefile]: Should we express the actuator image as an environment variable too? That way, if CI or anybody wants to test a different image we don't have to edit the file.

File: test/machines/machines_test.go [hunk 2]
```diff
@@ -191,7 +191,13 @@ var _ = framework.SigKubeDescribe("Machines", func() {
 			f.CreateClusterAndWait(cluster)
 
 			// Create master machine and verify the master node is ready
-			masterUserDataSecret := manifests.MasterMachineUserDataSecret("masteruserdatasecret", testNamespace.Name)
+			masterUserDataSecret, err := manifests.MasterMachineUserDataSecret(
+				"masteruserdatasecret",
+				testNamespace.Name,
+				[]string{"\\$(curl -s http://169.254.169.254/latest/meta-data/public-hostname)", "\\$(curl -s http://169.254.169.254/latest/meta-data/public-ipv4)"},
```
Comment (frobware) [inline test/machines/machines_test.go]: Why are we escaping here? Would a string `literal` suffice?

Comment (ingvagabund) [reply test/machines/machines_test.go]: It's used inside a template

**Comment by ingvagabund** [comment]
> Your commit says "WIP" - still so?

Currently testing https://github.com/openshift/cluster-api-actuator-pkg/pull/6 through this PR. Once green, I will merge it. The revendor `github.com/openshift/cluster-api-actuator-pkg` and rebase this PR.

**Comment by frobware** [comment]
> Then revendor `github.com/openshift/cluster-api-actuator-pkg` and rebase this PR.

OK, ping me when ready.

**Comment by bison** [comment]
Seems fine.

/lgtm

**Pull request merged**